### PR TITLE
Create new DK parser for exchanges

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
The old DK parser used a SQL query API that is no longer supported.
## Description
This PR switches to a new API for the exchanges and removed the production parser capabilities as they used non validated data prone to errors.

Fixes: #4567

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
@mathilde-daugy Could you take a look at this?
Would be great to get it merged so we get exchange data back for DK-DK*

File: parsers/DK.py [hunk 1]
```diff
@@ -1,250 +1,125 @@
...
-    "energy_bal": "02356e88-7c4e-4ee9-b896-275d217cc1b9",
+EXCHANGE_MAPPING = {
+    "DE->DK-DK1": {"id": "ExchangeGermany", "direction": 1, "priceArea": "DK1"},
+    "DK-DK1->DK-DK2": {"id": "ExchangeGreatBelt", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->NL": {"id": "ExchangeNetherlands", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->NO-NO2": {"id": "ExchangeNorway", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->SE-SE3": {"id": "ExchangeSweden", "direction": -1, "priceArea": "DK1"},
+    "DK-DK2->SE-SE4": {"id": "ExchangeSweden", "direction": -1, "priceArea": "DK2"},
+    "DK-BHM->SE-SE4": {"id": "BornholmSE4", "direction": -1, "priceArea": "DK2"},
 }
```
Comment (mathilde-daugy) [inline parsers/DK.py:18]: is DE->DK-DK2 data not available? it is in our exchanges.json and also calls DK.fetch_exchange

Comment (VIKTORVAV99) [reply parsers/DK.py:18]: I must have missed that!
I'll add it and check if it has data right away.

Comment (VIKTORVAV99) [reply parsers/DK.py:19]: Added `DE->DK-DK2`so this should be resolved now. Nice catch.

Comment (mathilde-daugy) [reply parsers/DK.py:19]: ok thanks Viktor, I will test and approve :)

**Pull request merged**